src/app/services/map.service.ts

```diff
@@ -755,9 +769,8 @@ export class MapService {
       'paint': {
         'circle-color': '#9bbcf2', 'circle-opacity': 0.2, 'circle-radius': 0,
         'circle-stroke-width': 1, 'circle-stroke-color': '#9bbcf2', 'circle-stroke-opacity': 0.5,
-        'circle-radius-transition': { 'duration': 0 }
+        // 'circle-radius-transition': { 'duration': 0 }  # FIXME @dotcs: According to types, this property is not defined

```

Comment:
@Binnette Do you know if this `circle-radius-transition` had an effect in the UI? I'm wondering because the property should not exist according to the types. Is this an error in the types or in the OsmGo codebase?

Reply:
https://maplibre.org/maplibre-gl-js-docs/style-spec/transition/
circle-radius-transition exist in style specification, this is an error in the types

Reply:
I remember why there is a duration: 300.
It represents the GPS accuracy in meters, but mapbbox GL js only accepts pixels.
So at each zoom, there is a recalculation of the radius to convert meters to pixels. That's why the tansition = 0 is important (default is 300ms)

https://maplibre.org/maplibre-gl-js-docs/style-spec/layers/#paint-circle-circle-radius